src/runtime/FunctionObject.cpp

```diff
@@ -30,7 +30,7 @@
 
 namespace Escargot {
 
-void FunctionObject::initStructureAndValues(ExecutionState& state, bool isConstructor, bool isGenerator)
+void FunctionObject::initStructureAndValues(ExecutionState& state, bool isConstructor, bool isGenerator, bool isAsync)
```

Comment:
`isAsync` parameter is not used in this function too. Is this parameter really necessary?

Reply:
I make the parameter because it can be used for AsyncGeneratorFunction